src/SMG/ManagerBundle/Controller/ManagersController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace SMG\ManagerBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use FOS\RestBundle\Controller\Annotations;
+use SMG\UserBundle\Entity\User;
+use FOS\RestBundle\View\View;
+use FOS\RestBundle\Controller\FOSRestController;
+use Symfony\Component\HttpFoundation\Response;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+
+class ManagersController extends FOSRestController
+{
+    use Traits\TokenFromHeaderTrait;
+    use Traits\HandleUserTrait;
+
+    /**
+     * TODO: annotation actually required to make difference
+     * with the other API which has the same name
+     *
+     * @Annotations\Post("/admin/users")
+     *
+     * @ParamConverter("user", converter="fos_rest.request_body")
+     */
+    public function postUserAction(User $user)
+    {
+        $this->throwIfClientNot('backend');
+        $manager = $this->get('fos_user.user_manager');
+
+        // TODO: refactor common parameters
+        // settings in the HandleUserTrait.
+        $token = $manager->deleteIfNonEnabledExists($user);
+
+        $errors = $this->validates($user);
+        if (count($errors) > 0) {
+            return $this->handleView(
+                new View($errors, Response::HTTP_BAD_REQUEST)
+            );
+        }
+
+        $newUser = $manager->createUser();
+
+        $phoneNumber = $user->getPhoneNumber();
+        $email = $user->getEmail();
+
+        if (!is_null($phoneNumber)) {
+            $phoneNumber = str_replace('+', '00', $phoneNumber);
+        }
+
+        $newUser->setPhoneNumber($phoneNumber);
+        $newUser->setEmail($email);
+        $newUser->setUsername($user->getUsername());
+        $newUser->setPlainPassword($user->getPlainPassword());
+        $newUser->setRoles($user->getRoles());
+        $newUser->setEnabled(true);
+        $newUser->setLocked(true);
+        $manager->updateUser($newUser);
+
+        return $this->handleView(
```

Comment:
we should return the id of the created user,  otherwise how can you link to it ?